### ISSUE TITLE
Degradation of Certain Fields After Language Translation

### ISSUE THREAD

**Issue by enesphp** [orangehrm/orangehrm#1737]
### OrangeHRM version

5.5

### Environment details

- OrangeHRM source: Cpanel Softaclous
- Platform: CloudLinux v8.8.0
- PHP version: 7.4
- Database and version:  MySQL  8.0.34  -  libmysql - mysqlnd 8.1.16
- Web server: LiteSpeed 
- Browser: Chrome Last Version

### Describe the bug

So I am trying to translate OrangeHRM to Turkish and I followed the method that was put by OrangeHRM and when I partially translated it and wanted to try it out I put the localization to Turkish and it started giving out some errors in buttons and and translated texts such as shown below:
"buzz.buzz_newsfeed,  buzz.post, claim.employee_claims,  claim.assign_claim"

![image](https://github.com/orangehrm/orangehrm/assets/29520460/f9b48142-1d6c-4e1b-b1c2-f2b765f0288f)

![image](https://github.com/orangehrm/orangehrm/assets/29520460/6c089f09-54b3-44c0-bdd3-c91671e2adc9)

![image](https://github.com/orangehrm/orangehrm/assets/29520460/4688e52e-6dc3-44e0-b5e6-1aaedc53687a)

**Comment by enesphp** [comment]
Console output on the page where I got the error
![image](https://github.com/orangehrm/orangehrm/assets/29520460/71aac1e7-c988-4782-a8cc-54b1cadc8908)
![image](https://github.com/orangehrm/orangehrm/assets/29520460/05de9eff-6455-4538-b619-435c60a8f9df)
![image](https://github.com/orangehrm/orangehrm/assets/29520460/8b740702-d233-434a-aad2-0f7e0be1c3f1)

**Comment by enesphp** [comment]
I have solved the issue. The root cause of the problem occurs when we copy the translated text from the translation screen and paste it into the input field, then click "save."

Here's what I have identified:

I queried the database with the corresponding Turkish Language ID in the table where translations are stored. "SELECT * FROM ohrm_i18n_translate WHERE language_id = 480;" Turkish Language ID number is 480.

Then, I exported all the translations I made for the Turkish language in SQL format, opened it in a text editor. I noticed that some data contains hidden invalid characters like <80a00>. My guess is that these characters might have come from copy-pasting during translation. When we save the translated text with unnecessary characters like <80a00> into the input field, it gets stored in the database and caches are updated.

As a solution, I applied the following code in the database: "DELETE FROM ohrm_i18n_translate WHERE language_id=480;" With this code, I deleted all the Turkish content for which I had done translations.

I reviewed the downloaded SQL file, removed the lines containing faulty data, re-imported the SQL, and deleted the src/cache/orangehrm folder via FTP.

The system is now functioning properly.
